catkit/hardware/iris_ao/segmented_dm_command.py

```diff
@@ -452,7 +464,7 @@ def plot_psf(self, rotation_angle, figure_name_prefix, out_dir, vmin=10e-8,
 
 
 def load_command(segment_values, dm_config_id, wavelength, testbed_config_id,
-                 apply_flat_map=True):
+                 apply_flat_map=True, filename_flat=''):
```

Comment:
Since `filename_flat` is only used when `apply_flat_map = True`, I recommend changing the default to `None`